canonical/react-components
# refactor: migrate from MDX to CSF stories with autodocs

## Done

This PR migrates Card component stories from MDX to CSF, if accepted I will do it with all remaining components

- Created a CSF file for rendering stories of the card component
- Allowed [autodocs](https://storybook.js.org/docs/writing-docs/autodocs) on it
- Deleted MDX version of Card's stories

## Screenshots


https://github.com/canonical/react-components/assets/16295402/9cb0dd0f-146d-47bf-b6c1-371fc0d37818


## QA

### QA steps

- Run the project
- Open Card stories
- Check that the behaviors haven't changed
- Open other stories
- Check that they didn't break as well
- Open the code
- Check that there is less code, less cognitive overload and it has improved

## Fixes

Fixes: #1001  .
